contribute.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "django-browserid",
+    "description": "django-browserid is a library that integrates BrowserID authentication into Django.",
```

Comment:
@bobsilverberg I think Persona is the Mozilla **instance** of the technology called "BrowserID".